docs/user-guide/deployments-administration/maintenance/recovery-mode.md

```diff
@@ -0,0 +1,58 @@
+---
+keywords: [GreptimeDB, recovery mode, metasrv, maintenance]
+description: Guide for using GreptimeDB cluster recovery mode to address datanode startup failures and recover from region data loss or corruption.
+---
+
+# Cluster Recovery Mode
+
+Recovery mode is a safety feature in GreptimeDB that allows users to manually recover the cluster from a failed state.
```

Comment:
```suggestion
Recovery mode is a safety feature in GreptimeDB that allows developers to manually recover the cluster from a failed state.
```

Reply:
resolved in[f543376](https://github.com/GreptimeTeam/docs/pull/2013/commits/f5433761bb8e74f494210ba766397ed863bec0e9)